app/views/posts/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>All posts page of a given user</h1>
\ No newline at end of file
+<%= render partial: 'partials/user_card', locals: { user: @user } %>
+
+<% @posts.each do |post| %>
+  <div class='post-ctr'>
+    <%= render partial: 'partials/post', locals: { post: post, short: true } %>
+    <% if post.comments.length > 0 %>
+      <div class='comments'>
+        <% post.five_recent_comments.each do |comment| %>
+          <%= render partial: 'partials/comment', locals: { comment: comment } %>
+        <% end %>
+      </div>
+    <% end %>
+    <%= link_to "/users/#{@user.id}/posts/#{post.id}" do %>
+      <button type='button'>See this post</button>
+    <% end %>
```

Comment:
- [ ] Post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). Can you implement it?